### ISSUE TITLE
Suspended users shown into the "Pending" ones

### ISSUE THREAD

**Issue by Zsavajji** [craftcms/cms#10383]
### Steps to reproduce

1.
2.

### Additional info

- Craft version:
- PHP version:
- Database driver & version:
- Plugins & versions:

**Comment by brandonkelly** [comment]
This is likely because they are both pending and suspended. Go to **Settings** → **Users** → **Settings**. Is the “Suspend users by default” setting enabled?

**Comment by Zsavajji** [comment]
Yes, it's indeed enabled, since i need it to implement a registration approval flow.
Shouldn't the filter just show the latest state of the user though? It's a bit confusing this way.

Thank you!

**Comment by brandonkelly** [comment]
Agree it’s a little confusing, but it felt like a bug that pending-and-suspended users weren’t getting included, since they are in fact pending.

**Comment by Zsavajji** [comment]
I understand, but the user itself shows "Suspended" as a state, and i think it's a bit unclear for the end user to see users grouped under a status they do not show in any other meaningful way.
This visualization also make the process of this manual user approval workflow less straightforward (to me at least). 
Can a settings toggle or config option be a better idea? by letting the user choose you could have the "best of both worlds"
